client/packages/editor/src/components/widgets/table/header/TableHeader.css

```diff
@@ -0,0 +1,11 @@
+.table .table-column-header {
+  padding: 10px;
+  text-align: start;
+  font-size: 10px;
```

Comment:
maybe we should avoid absolute values in a concrete widget ... making the inscription editor easier to access on any device ... even in SWT the user could choose the font-size for the editors. If we go back to static values, we will loose users with bad eye-sight :)

Reply:
yes I need to change this px values to rem or to another responsive unit :)